changeset.go

```diff
@@ -1870,6 +1959,38 @@ func (cs *Changeset) upsertPriorityClass(
 	})
 }
 
+func (cs *Changeset) upsertServiceMonitor(ctx context.Context, tr *ChangesetResource, data []byte) (*ChangesetResource, error) {
+	monitor, err := ParseServiceMonitor(bytes.NewReader(data))
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
+	log := log.WithFields(log.Fields{
+		"cs":              tr.String(),
+		"service_monitor": fmt.Sprintf("%v/%v", monitor.Namespace, monitor.Name),
+	})
+	log.Infof("upsert service monitor %v", formatMeta(monitor.ObjectMeta))
```

Comment:
nitpick: As I remember by our conventions log lines should start from capital letter.

Reply:
Also, by the way. From the output of update hooks I don't see any logs like this one. There is somewhere a problem.

Reply:
Yeah, all existing log entries in rigging don't follow this convention so I just wanted to keep it consistent. Also, I actually do see these logs in the update hook. Not sure why you don't see them.